docs/topics/proc_using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+
+link:https://docs.openrewrite.org/[OpenRewrite], which is integrated into the {ProductShortName} CLI, uses recipes to automate large-scale, distributed source code refactoring.
+
+You can use OpenRewrite in the {ProductShortName} CLI to prepare Java applications for migration.
+
+[IMPORTANT]
+====
+OpenRewrite recipe support is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
+The `JavaxtoJakarta` recipe renames imported `javax` packages to their `jakarta` equivalents.
+
+.Procedure
+
+. Specify the OpenRewrite recipe when you run `mta-cli`:
++
+[source,terminal]
+----
+$ ./mta-cli --openrewrite "-DactiveRecipes=<recipe name>"  \
+  --input /path/to/source/project --goal dryRun
+----
+
+* `--openrewrite`: Flag specifying to run an OpenRewrite migration, instead of an MTA analysis.
+
+* `"-DactiveRecipes=<recipe name>"` : The OpenRewrite recipe to apply to the input project. `org.jboss.windup.JavaxtoJakarta` is the default shipped recipe but you can add your own to the shipped `rewrite.yml` and run that recipe instead. The `rewrite.yml file` is located in the `rules/openrewrite/`` folder in the uncompressed MTA distribution.
```

Comment:
It's not clear from this description what the user is supposed to use as the "recipe name":

JavaxtoJakarta? 
org.jboss.windup.JavaxtoJakarta? 
rewrite.yml?

Reply:
I've asked Mark to double-check the revised code block. I removed the sentence that includes the location of the .yml file. Should I rewrite the last part of the description as "you can add your own to the shipped `.yml` file." ?

Reply:
If the path to the yml file isn't being included I don't think we should mention creating your own recipes at all because the user won't know where to put them.